source/module_io/test/parameter_pool_test.cpp

```diff
@@ -0,0 +1,396 @@
+// #include "gtest/gtest.h"
```

Comment:
What is the usage of this file?

Reply:
This test code is for the formal access test of the module in the next stage. It cannot be used before the new input module is perfected, so it is not useful for the time being.